### ISSUE TITLE
move json schema to main repo 

### ISSUE THREAD

**Issue by maxandersen** [jbangdev/jbang-idea#4]
I think https://github.com/jbangdev/jbang-idea/blob/main/src/main/resources/jbang-catalog-schema.json should be in the main repo and published to schemastore.org so it works for other IDE's json editors not just jbang-idea.

**Comment by linux-china** [comment]
Yes, I think it's better to move it to jbang.dev repo, and all editors can use this schema file from  http://jbang.dev/schema.json  for code completion.

**Comment by linux-china** [comment]
@maxandersen  could you make a review for  https://github.com/jbangdev/jbang-idea/blob/main/src/main/resources/jbang-catalog-schema.json  then we can submit a PR to https://github.com/SchemaStore/schemastore

**Comment by maxandersen** [comment]
Open the pr on jbang then we'll get that reviewed. I don't expect an issue.
